src/test/scala/SystemTests.scala

```diff
@@ -94,8 +94,8 @@ trait SystemTests extends AnyFunSuite {
     val failureMsg = if timedOut then "SMT Solver timed out" else
       (verified, shouldVerify, xor(verified, proveFailed)) match {
         case (true, true, true) => "Test passed"
-        case (false , false, true) => "Test passed"
-        case (_, _, false) => "Prover error: unknown result: " + boogieResult
+        case (false, false, true) => "Test passed"
+        case (_, _, false) => "Prover error: unknown result"
```

Comment:
Why change this, this was useful

Reply:
This was probably unintentionally removed in a merge